cypress/support/pages/contactForm.pageObject.js

```diff
@@ -19,11 +19,11 @@ class ContactFormPageObject extends PageObject {
     return cy.contains('.btn', 'Send message');
   }
 
-  typeEmail(email) {
+  typeEmail() {
     this.emailField.type(email, { force: true });
```

Comment:
The function 'typeEmail' is missing a parameter for the email value. You should pass an 'email' parameter to this function and use it in the 'type' method.